geo/src/algorithm/convert_angle_unit.rs

```diff
@@ -0,0 +1,83 @@
+use geo_types::Coord;
+use geo_types::CoordFloat;
+
+use crate::{MapCoords, MapCoordsInPlace};
+
+pub trait ToRadians<T: CoordFloat>: Sized + MapCoords<T, T, Output = Self> {
+    fn to_radians(&self) -> Self {
+        self.map_coords(|Coord { x, y }| Coord {
+            x: x.to_radians(),
+            y: y.to_radians(),
+        })
+    }
+}
+impl<T: CoordFloat, G: MapCoords<T, T, Output = Self>> ToRadians<T> for G {}
+
+pub trait ToRadiansInPlace<T: CoordFloat>: MapCoordsInPlace<T> {
+    fn to_radians_in_place(&mut self) {
```

Comment:
Can you put this method in `ToRadians` and get rid of the `ToRadiansInPlace` trait?

e.g. here's how we do it with the [`AffineOps`](https://github.com/georust/geo/blob/main/geo/src/algorithm/affine_ops.rs#L47) trait:

```
impl<T: CoordNum, M: MapCoordsInPlace<T> + MapCoords<T, T, Output = Self>> AffineOps<T> for M
```

My motivation is that people looking for one method might actually want the other and IMO it's easier to find when they're in the same trait. 

For the same reason, I wish we could merge `MapCoords` and `MapCoordsInPlace` but it gets weird because `MapCoordsInPlace` methods don't utilize the NT in `MapCoords<_, NT>`.

Reply:
Fixed :)